### PR TITLE
Add TemporalConvolution markdown

### PR DESCRIPTION
**Pull request by psyyz10**
## What changes were proposed in this pull request?

Add TemporalConvolution markdown

## Related links or issues (optional)
fixed https://github.com/intel-analytics/BigDL/issues/1338

### CODE REVIEW

File: docs/docs/APIdocs/Layers/Convolution-Layers.md [hunk 1]
```diff
@@ -887,3 +887,112 @@ Output is
...
+**Python:**
+```python
+module = TemporalConvolution(
+  input_frame_size, output_frame_size, kernel_w, stride_w = 1, propagate_back = True,
+  w_regularizer = None, b_regularizer = None, init_weight = None, init_bias = None,
+  init_grad_weight = None, init_grad_bias = None
+  )
+```
+
+Applies a 1D convolution over an input sequence composed of nInputFrame frames..
```
Comment (i8run) [inline docs/docs/APIdocs/Layers/Convolution-Layers.md]: double .

File: docs/docs/APIdocs/Layers/Convolution-Layers.md [hunk 2]
```diff
@@ -887,3 +887,112 @@ Output is
...
+
+ * @param inputFrameSize The input frame size expected in sequences given into `forward()`.
+ * @param outputFrameSize The output frame size the convolution layer will produce.
+ * @param kernelW The kernel width of the convolution
+ * @param strideW The step of the convolution in the width dimension.
+ * @param propagateBack Whether propagate gradient back, default is true.
+ * @param wRegularizer instance of `Regularizer`
+                     (eg. L1 or L2 regularization), applied to the input weights matrices.
+ * @param bRegularizer instance of `Regularizer`
+                     applied to the bias.
```
Comment (i8run) [inline docs/docs/APIdocs/Layers/Convolution-Layers.md]: shoud be in a line

File: docs/docs/APIdocs/Layers/Convolution-Layers.md [hunk 3]
```diff
@@ -887,3 +887,112 @@ Output is
...
+(`nInputFrame` x `inputFrameSize`) or a 3D tensor
+(`nBatchFrame` x `nInputFrame` x `inputFrameSize`).
+
+ * @param inputFrameSize The input frame size expected in sequences given into `forward()`.
+ * @param outputFrameSize The output frame size the convolution layer will produce.
+ * @param kernelW The kernel width of the convolution
+ * @param strideW The step of the convolution in the width dimension.
+ * @param propagateBack Whether propagate gradient back, default is true.
+ * @param wRegularizer instance of `Regularizer`
+                     (eg. L1 or L2 regularization), applied to the input weights matrices.
```
Comment (i8run) [inline docs/docs/APIdocs/Layers/Convolution-Layers.md]: should be in a line

File: docs/docs/APIdocs/Layers/Convolution-Layers.md [hunk 4]
```diff
@@ -887,3 +887,112 @@ Output is
...
+                     applied to the bias.
+ * @param initWeight Initial weight
+ * @param initBias Initial bias
+ * @param initGradWeight Initial gradient weight
+ * @param initGradBias Initial gradient bias
+ * @tparam T The numeric type in the criterion, usually which are `Float` or `Double`
+ 
+**Scala example:**
+```scala
+import com.intel.analytics.bigdl.numeric.NumericDouble
```
Comment (shane-huang) [inline docs/docs/APIdocs/Layers/Convolution-Layers.md]: Use Float

File: docs/docs/APIdocs/Layers/Convolution-Layers.md [hunk 5]
```diff
@@ -887,3 +887,112 @@ Output is
...
+Random.setSeed(seed)
+val input = Tensor(10, 5).apply1(e => Random.nextDouble())
+val gradOutput = Tensor(3, 3).apply1(e => Random.nextDouble())
+
+val output = layer.updateOutput(input)
+println(output)
+-0.46563127839495466	0.013695616322773968	-0.43273799278822744
+-0.1533250673179169	-0.04438171389050036	-0.7511583415829347
+-0.6131689030252309	-0.050553649561625946	-0.9389273227418238
+  [com.intel.analytics.bigdl.tensor.DenseTensor of size 3x3]
```
Comment (i8run) [inline docs/docs/APIdocs/Layers/Convolution-Layers.md]: output should not in the code

File: docs/docs/APIdocs/Layers/Convolution-Layers.md (hunk 2 above)
Comment (psyyz10) [inline docs/docs/APIdocs/Layers/Convolution-Layers.md]: will be one line in markdown view

File: docs/docs/APIdocs/Layers/Convolution-Layers.md (hunk 3 above)
Comment (psyyz10) [inline docs/docs/APIdocs/Layers/Convolution-Layers.md]: will be one line in markdown view

**Pull request merged**